### PR TITLE
Support latest build_web_compilers across all packages

### PR DESCRIPTION
**Pull request by kevmoo**

### CODE REVIEW

**Comment by jakemac53** [comment]
You could just update the test expectations, this is a difference between ddc and ddk. I filed https://github.com/dart-lang/sdk/issues/36736 to track.

**Comment by kevmoo** [comment]
> You could just update the test expectations, this is a difference between ddc and ddk. I filed [dart-lang/sdk#36736](https://github.com/dart-lang/sdk/issues/36736) to track.

ack!

File: dwds/test/chrome_proxy_service_test.dart [hunk 1]
```diff
@@ -183,14 +182,17 @@ void main() {
...
+      expect(isolate.rootLib.uri, endsWith('main.dart'));
+
       expect(
           isolate.libraries,
           containsAll([
-            predicate((LibraryRef lib) => lib.uri == 'dart:core'),
-            predicate((LibraryRef lib) => lib.uri == 'dart:html'),
-            predicate((LibraryRef lib) => lib.uri == 'package:path/path.dart'),
-            predicate((LibraryRef lib) => lib.uri == 'hello_world/main.dart'),
+            _libRef('dart:core'),
```
Comment (kevmoo) [inline dwds/test/chrome_proxy_service_test.dart]: @jakemac53 – `TypeMatcher` is the bomb!

File: webdev/pubspec.yaml [hunk 2]
```diff
@@ -37,12 +37,12 @@ dev_dependencies:
...
   build_runner: ^1.0.0
   built_value_generator: ^6.4.0
-  build_web_compilers: ^1.0.0
+  build_web_compilers: '>=1.0.0 <3.0.0'
   build_verify: ^1.0.0
   build_version: ^2.0.0
   graphs: ^0.2.0
   js: ^0.6.1
-  test: ^1.0.0
+  test: ^1.6.0
```
Comment (kevmoo) [inline webdev/pubspec.yaml]: Dep w/ `isA` function

**Pull request merged**